extensions/wikia/CommunityPage/models/CommunityPageSpecialInsightsModel.class.php

```diff
@@ -61,7 +61,7 @@ private function getInsightModule( $type, $sortingType = self::INSIGHTS_MODULE_S
 		$insightPages['title'] = wfMessage( 'communitypage-' . $type. '-title' )->text();
 		$insightPages['description'] =  wfMessage( 'communitypage-' . $type. '-description' )->text();
 
-		$insightPages['fulllistlink'] = SpecialPage::getTitleFor( 'Insights', $type )->getLocalURL();
+		$insightPages['fulllistlink'] = SpecialPage::getTitleFor( 'Insights', $type )->getLocalURL() . "?sort=" . self::INSIGHTS_MODULE_SORT_TYPE;
```

Comment:
You can pass url parameters as an array to `getLocalURL()` method

see: https://github.com/Wikia/app/blob/dev/includes/Title.php#L1442